code/modules/organs/organ.dm

```diff
@@ -259,6 +259,10 @@ var/list/organ_cache = list()
 	min_bruised_damage = 15
 	min_broken_damage = 35
 
+/obj/item/organ/proc/digitize() //Used to make circuit organs like robobrains
+	robotize()
+	src.status &= "digital"
```

Comment:
&= is also not what you want, that's a bitwise operation. If you were trying to flip the digital status bit you would need |= and if you were just straight up adding it you would need +=.